js/read_dir_test.ts

```diff
@@ -22,12 +22,12 @@ function assertSameContent(files: FileInfo[]) {
   assertEqual(counter, 2);
 }
 
-testPerm({ write: true, read: true }, function readDirSyncSuccess() {
+testPerm({ read: true }, function readDirSyncSuccess() {
   const files = deno.readDirSync("tests/");
   assertSameContent(files);
 });
 
-testPerm({ write: true, read: false }, function readDirSyncPerm() {
+testPerm({ read: false }, function readDirSyncPerm() {
```

Comment:
Why is this needed?

Reply:
that's a good point, it's not necessary. the only justification I can think of is that it declares to the next person reading the code which permission is lacking.